### DISCUSSION TITLE
Welcome to kindle-cover-fixer Discussions!

### DISCUSSION THREAD

**Discussion by weto91** [weto91/kindle-cover-fixer#1] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by robinexist** [comment]
i dont know how to run your file

**Reply by weto91** [reply]
what file?

**Comment by Drorex11** [comment]
This app seems AMAZING but I can't use it because I only have a MacBook. Is there any way it can be compatible with apple?

**Reply by weto91** [reply]
Sorry, it is only supported on Windows. You can use Parallels to virtualize windows.

**Comment by SWausHH** [comment]
Hi app seems to be nice BUT it always crashes when I unse the FIND buttom. (Kindle Library (Calibre) and Kindle attached.
Covers are flipping on the rright side from Library.

Bit after pressing FIND Buttom the app closed. 

Any Idea?

BR Stephan

**Reply by weto91** [reply]
Hello, I need you to give me more information:
-What Kindle do you have?
- Do you have a book title with strange characters (Asian characters, symbols...)
- Can you give me a screenshot of the app before it crashed?
- A log is generated in the installation folder. Can you send it to me?

**Comment by SWausHH** [comment]
Hi,
thnks for reply,



Nothave 6th paperwhite.
caracters of book are not "strange"
Which LOG File do you mean? I find only dll´s?

See attached screenshot.
the covers shown right are flipping. the kindle is connected. right after pushing "FIND" it breaks.

BR regards

Stephan

[cid:3e17436d-9456-49f8-afa7-650a09aa1346]

________________________________

**Reply by weto91** [reply]
Hi, the logfile is on your app folder. "[installation path] \Log\Kindle_Cover_Fixer.log"

**Comment by SWausHH** [comment]
Hi,

I can not find such a path.
I didnt install I just extract zip in one path an klick on appFile.
[cid:a11f76d9-6756-4ee9-9a37-5a62e2c65688]

________________________________

**Reply by weto91** [reply]
Im sorry, my mistake. The log is on %APPDATA%/Roaming/Kindle Cover Fixer/Log

You can open the file explorer in your windows and type %APPDATA%, then navigate to Roaming/Kindle Cover Fixer/Log. Here is the log file that i want.


Regards.

**Comment by SWausHH** [comment]
Hi,

I can not find such a path.
I didnt install I just extract zip in one path an klick on appFile.
[cid:a11f76d9-6756-4ee9-9a37-5a62e2c65688]

________________________________

**Comment by SWausHH** [comment]
hi,

so here we are with the log file.
maybe it helps analysing. 😉

________________________________

**Reply by weto91** [reply]
Sorry but i cant find the log, can you attach it?

**Comment by SWausHH** [comment]
[INFO] 04.03.2025 08:02:13 | Started at: 04.03.2025 08:02:13
[INFO] 04.03.2025 08:02:13 | OS version: Microsoft Windows NT 10.0.26100.0
[INFO] 04.03.2025 08:02:13 | Process ID: 23156
[INFO] 04.03.2025 08:02:13 | System memory page: 4096
[INFO] 04.03.2025 08:02:13 | Version: 6.0.36
[INFO] 04.03.2025 08:02:13 | KCF Path: C:\Users\Steph\Desktop\Kindle_Cover_Fixer_setup_2_6\Kindle Cover Fixer V2.exe
[INFO] 04.03.2025 08:02:13 | KCF Version: 2.6
[INFO] 04.03.2025 08:02:13 | #############################################################
[LIBRARY] 04.03.2025 08:02:14 | Library: C:/Users/Steph/Calibre-Bibliothek loaded.
[LIBRARY] 04.03.2025 08:02:14 | Library: C:/Users/Steph/OneDrive/1 - Dokumente/CAL_Test loaded.
[DEVICE] 04.03.2025 08:02:15 | Device connected or disconnected, new device list: Kindle (Other) |
[INFO] 04.03.2025 08:02:37 | Started at: 04.03.2025 08:02:37
[INFO] 04.03.2025 08:02:37 | OS version: Microsoft Windows NT 10.0.26100.0
[INFO] 04.03.2025 08:02:37 | Process ID: 23156
[INFO] 04.03.2025 08:02:37 | System memory page: 4096
[INFO] 04.03.2025 08:02:37 | Version: 6.0.36
[INFO] 04.03.2025 08:02:37 | KCF Path: C:\Users\Steph\Desktop\Kindle_Cover_Fixer_setup_2_6\Kindle Cover Fixer V2.exe
[INFO] 04.03.2025 08:02:37 | KCF Version: 2.6
[INFO] 04.03.2025 08:02:37 | #############################################################
[INFO] 04.03.2025 08:02:37 | The application is closing. Bye!


________________________________
